code-challenges.js

```diff
@@ -7,6 +7,50 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
+//create a function that takes in an array
+// const colorFilterShuffle = (array) => {
+// //shuffle all the items in the array
+//     for (let i = array.length - 1; i > 0; i--) {
+//         let j = Math.floor(Math.random() * (i + 1));
+//         [array[i], array[j]] = [array[j], array[i]];}
+//     //filter the original array, take in the value and the index
+//     return array.filter((value, index) => { 
+//     // in order to return the array with the first item removed
+//     return index > 0
+//     })
+// }
+
+// console.log(colorFilterShuffle(collections))
+
+//I was able to get the output to return the original array, shuffled, and without the first value, but I wasn't sure how to make this loop continuously so it always removes the first value until the array is empty.
```

Comment:
nice! the prompt didn't include it running on its own until the array was empty but I appreciate you trying to take it to that next level! look into recursion for this.